### DISCUSSION TITLE
When is the team planning to release macOS Ventura Virtual machines?

### DISCUSSION THREAD

**Discussion by SamarthKejriwal** [codemagic-ci-cd/codemagic-docs#1200] [Q&A] [answered]
Is there any timeline to release macOS venture based machines for CI/CD?

**Comment by icarusdust** [comment]
Hi @SamarthKejriwal, I have already discussed it with Denil. I will keep you posted.

**Comment by helinanever** [comment, marked as answer]
Hi @SamarthKejriwal, macOS Ventura build machines are already available: https://github.com/codemagic-ci-cd/codemagic-docs/discussions/1228
